src/NLog/Targets/Wrappers/DistinctlyThrottlingWrapperTarget.cs

```diff
@@ -31,10 +31,14 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
+#define NET3_5
```

Comment:
the NET3_5 etc is defined already in the csproj for the .NET 3.5 platform. If you set it here, everything is defined as net3_5..